Source/items.cpp

```diff
@@ -1201,7 +1201,7 @@ void GetItemPower(Item &item, int minlvl, int maxlvl, affix_item_type flgs, bool
 				continue;
 			if (onlygood && !ItemPrefixes[j].PLOk)
 				continue;
-			if (flgs == PLT_STAFF && ItemPrefixes[j].power.type == IPL_CHARGES)
+			if (HasAnyOf(flgs,AffixItemType::Staff) && ItemPrefixes[j].power.type == IPL_CHARGES)

```

Comment:
```suggestion
			if (HasAnyOf(flgs, AffixItemType::Staff) && ItemPrefixes[j].power.type == IPL_CHARGES)
```

Reply:
yup just noticed it. I wish Visual Studio would fix that before it uploads the commit...